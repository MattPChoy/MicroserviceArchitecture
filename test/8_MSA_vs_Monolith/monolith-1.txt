
          /\      |‾‾| /‾‾/   /‾‾/   
     /\  /  \     |  |/  /   /  /    
    /  \/    \    |     (   /   ‾‾\  
   /          \   |  |\  \ |  (‾)  | 
  / __________ \  |__| \__\ \_____/ .io

  execution: local
     script: -
     output: -

  scenarios: (100.00%) 1 scenario, 1000 max VUs, 2m30s max duration (incl. graceful stop):
           * default: 1000 looping VUs for 2m0s (gracefulStop: 30s)


running (0m00.8s), 1000/1000 VUs, 98 complete and 0 interrupted iterations
default   [   1% ] 1000 VUs  0m00.8s/2m0s

running (0m01.8s), 1000/1000 VUs, 281 complete and 0 interrupted iterations
default   [   1% ] 1000 VUs  0m01.8s/2m0s

running (0m02.8s), 1000/1000 VUs, 441 complete and 0 interrupted iterations
default   [   2% ] 1000 VUs  0m02.8s/2m0s

running (0m03.8s), 1000/1000 VUs, 600 complete and 0 interrupted iterations
default   [   3% ] 1000 VUs  0m03.8s/2m0s

running (0m04.8s), 1000/1000 VUs, 767 complete and 0 interrupted iterations
default   [   4% ] 1000 VUs  0m04.8s/2m0s

running (0m05.8s), 1000/1000 VUs, 927 complete and 0 interrupted iterations
default   [   5% ] 1000 VUs  0m05.8s/2m0s

running (0m06.8s), 1000/1000 VUs, 1090 complete and 0 interrupted iterations
default   [   6% ] 1000 VUs  0m06.8s/2m0s

running (0m07.8s), 1000/1000 VUs, 1250 complete and 0 interrupted iterations
default   [   6% ] 1000 VUs  0m07.8s/2m0s

running (0m08.8s), 1000/1000 VUs, 1405 complete and 0 interrupted iterations
default   [   7% ] 1000 VUs  0m08.8s/2m0s

running (0m09.8s), 1000/1000 VUs, 1577 complete and 0 interrupted iterations
default   [   8% ] 1000 VUs  0m09.8s/2m0s

running (0m10.8s), 1000/1000 VUs, 1739 complete and 0 interrupted iterations
default   [   9% ] 1000 VUs  0m10.8s/2m0s

running (0m11.8s), 1000/1000 VUs, 1919 complete and 0 interrupted iterations
default   [  10% ] 1000 VUs  0m11.8s/2m0s

running (0m12.8s), 1000/1000 VUs, 2080 complete and 0 interrupted iterations
default   [  11% ] 1000 VUs  0m12.8s/2m0s

running (0m13.8s), 1000/1000 VUs, 2242 complete and 0 interrupted iterations
default   [  11% ] 1000 VUs  0m13.8s/2m0s

running (0m14.8s), 1000/1000 VUs, 2405 complete and 0 interrupted iterations
default   [  12% ] 1000 VUs  0m14.8s/2m0s

running (0m15.8s), 1000/1000 VUs, 2567 complete and 0 interrupted iterations
default   [  13% ] 1000 VUs  0m15.8s/2m0s

running (0m16.8s), 1000/1000 VUs, 2726 complete and 0 interrupted iterations
default   [  14% ] 1000 VUs  0m16.8s/2m0s

running (0m17.8s), 1000/1000 VUs, 2893 complete and 0 interrupted iterations
default   [  15% ] 1000 VUs  0m17.8s/2m0s

running (0m18.8s), 1000/1000 VUs, 3057 complete and 0 interrupted iterations
default   [  16% ] 1000 VUs  0m18.8s/2m0s

running (0m19.8s), 1000/1000 VUs, 3234 complete and 0 interrupted iterations
default   [  16% ] 1000 VUs  0m19.8s/2m0s

running (0m20.8s), 1000/1000 VUs, 3415 complete and 0 interrupted iterations
default   [  17% ] 1000 VUs  0m20.8s/2m0s

running (0m21.8s), 1000/1000 VUs, 3598 complete and 0 interrupted iterations
default   [  18% ] 1000 VUs  0m21.8s/2m0s

running (0m22.8s), 1000/1000 VUs, 3770 complete and 0 interrupted iterations
default   [  19% ] 1000 VUs  0m22.8s/2m0s

running (0m23.8s), 1000/1000 VUs, 3933 complete and 0 interrupted iterations
default   [  20% ] 1000 VUs  0m23.8s/2m0s

running (0m24.8s), 1000/1000 VUs, 4094 complete and 0 interrupted iterations
default   [  21% ] 1000 VUs  0m24.8s/2m0s

running (0m25.8s), 1000/1000 VUs, 4254 complete and 0 interrupted iterations
default   [  21% ] 1000 VUs  0m25.8s/2m0s

running (0m26.8s), 1000/1000 VUs, 4412 complete and 0 interrupted iterations
default   [  22% ] 1000 VUs  0m26.8s/2m0s

running (0m27.8s), 1000/1000 VUs, 4573 complete and 0 interrupted iterations
default   [  23% ] 1000 VUs  0m27.8s/2m0s

running (0m28.8s), 1000/1000 VUs, 4733 complete and 0 interrupted iterations
default   [  24% ] 1000 VUs  0m28.8s/2m0s

running (0m29.8s), 1000/1000 VUs, 4934 complete and 0 interrupted iterations
default   [  25% ] 1000 VUs  0m29.8s/2m0s

running (0m30.8s), 1000/1000 VUs, 5166 complete and 0 interrupted iterations
default   [  26% ] 1000 VUs  0m30.8s/2m0s

running (0m31.8s), 1000/1000 VUs, 5380 complete and 0 interrupted iterations
default   [  26% ] 1000 VUs  0m31.8s/2m0s
