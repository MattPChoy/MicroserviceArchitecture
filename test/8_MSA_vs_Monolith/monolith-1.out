
          /\      |‾‾| /‾‾/   /‾‾/   
     /\  /  \     |  |/  /   /  /    
    /  \/    \    |     (   /   ‾‾\  
   /          \   |  |\  \ |  (‾)  | 
  / __________ \  |__| \__\ \_____/ .io

  execution: local
     script: loadtest.js
     output: json (monolith-1.json)

  scenarios: (100.00%) 1 scenario, 1000 max VUs, 2m30s max duration (incl. graceful stop):
           * default: 1000 looping VUs for 2m0s (gracefulStop: 30s)


running (0m00.7s), 1000/1000 VUs, 98 complete and 0 interrupted iterations
default   [   1% ] 1000 VUs  0m00.7s/2m0s

running (0m00.9s), 1000/1000 VUs, 134 complete and 0 interrupted iterations
default   [   1% ] 1000 VUs  0m00.9s/2m0s

running (0m01.9s), 1000/1000 VUs, 292 complete and 0 interrupted iterations
default   [   2% ] 1000 VUs  0m01.9s/2m0s

running (0m02.9s), 1000/1000 VUs, 458 complete and 0 interrupted iterations
default   [   2% ] 1000 VUs  0m02.9s/2m0s

running (0m03.9s), 1000/1000 VUs, 624 complete and 0 interrupted iterations
default   [   3% ] 1000 VUs  0m03.9s/2m0s

running (0m04.9s), 1000/1000 VUs, 789 complete and 0 interrupted iterations
default   [   4% ] 1000 VUs  0m04.9s/2m0s

running (0m05.9s), 1000/1000 VUs, 955 complete and 0 interrupted iterations
default   [   5% ] 1000 VUs  0m05.9s/2m0s

running (0m06.9s), 1000/1000 VUs, 1115 complete and 0 interrupted iterations
default   [   6% ] 1000 VUs  0m06.9s/2m0s

running (0m07.9s), 1000/1000 VUs, 1298 complete and 0 interrupted iterations
default   [   7% ] 1000 VUs  0m07.9s/2m0s

running (0m08.9s), 1000/1000 VUs, 1467 complete and 0 interrupted iterations
default   [   7% ] 1000 VUs  0m08.9s/2m0s

running (0m09.9s), 1000/1000 VUs, 1633 complete and 0 interrupted iterations
default   [   8% ] 1000 VUs  0m09.9s/2m0s

running (0m10.9s), 1000/1000 VUs, 1807 complete and 0 interrupted iterations
default   [   9% ] 1000 VUs  0m10.9s/2m0s

running (0m11.9s), 1000/1000 VUs, 1969 complete and 0 interrupted iterations
default   [  10% ] 1000 VUs  0m11.9s/2m0s

running (0m12.9s), 1000/1000 VUs, 2126 complete and 0 interrupted iterations
default   [  11% ] 1000 VUs  0m12.9s/2m0s

running (0m13.9s), 1000/1000 VUs, 2286 complete and 0 interrupted iterations
default   [  12% ] 1000 VUs  0m13.9s/2m0s

running (0m14.9s), 1000/1000 VUs, 2466 complete and 0 interrupted iterations
default   [  12% ] 1000 VUs  0m14.9s/2m0s

running (0m15.9s), 1000/1000 VUs, 2644 complete and 0 interrupted iterations
default   [  13% ] 1000 VUs  0m15.9s/2m0s

running (0m16.9s), 1000/1000 VUs, 2814 complete and 0 interrupted iterations
default   [  14% ] 1000 VUs  0m16.9s/2m0s

running (0m17.9s), 1000/1000 VUs, 2977 complete and 0 interrupted iterations
default   [  15% ] 1000 VUs  0m17.9s/2m0s

running (0m18.9s), 1000/1000 VUs, 3160 complete and 0 interrupted iterations
default   [  16% ] 1000 VUs  0m18.9s/2m0s

running (0m19.9s), 1000/1000 VUs, 3324 complete and 0 interrupted iterations
default   [  17% ] 1000 VUs  0m19.9s/2m0s

running (0m20.9s), 1000/1000 VUs, 3490 complete and 0 interrupted iterations
default   [  17% ] 1000 VUs  0m20.9s/2m0s

running (0m21.9s), 1000/1000 VUs, 3661 complete and 0 interrupted iterations
default   [  18% ] 1000 VUs  0m21.9s/2m0s

running (0m22.9s), 1000/1000 VUs, 3822 complete and 0 interrupted iterations
default   [  19% ] 1000 VUs  0m22.9s/2m0s

running (0m23.9s), 1000/1000 VUs, 3982 complete and 0 interrupted iterations
default   [  20% ] 1000 VUs  0m23.9s/2m0s

running (0m24.9s), 1000/1000 VUs, 4161 complete and 0 interrupted iterations
default   [  21% ] 1000 VUs  0m24.9s/2m0s

running (0m25.9s), 1000/1000 VUs, 4340 complete and 0 interrupted iterations
default   [  22% ] 1000 VUs  0m25.9s/2m0s

running (0m26.9s), 1000/1000 VUs, 4504 complete and 0 interrupted iterations
default   [  22% ] 1000 VUs  0m26.9s/2m0s

running (0m27.9s), 1000/1000 VUs, 4703 complete and 0 interrupted iterations
default   [  23% ] 1000 VUs  0m27.9s/2m0s

running (0m28.9s), 1000/1000 VUs, 4893 complete and 0 interrupted iterations
default   [  24% ] 1000 VUs  0m28.9s/2m0s

running (0m29.9s), 1000/1000 VUs, 5063 complete and 0 interrupted iterations
default   [  25% ] 1000 VUs  0m29.9s/2m0s

running (0m30.9s), 1000/1000 VUs, 5233 complete and 0 interrupted iterations
default   [  26% ] 1000 VUs  0m30.9s/2m0s

running (0m31.9s), 1000/1000 VUs, 5395 complete and 0 interrupted iterations
default   [  27% ] 1000 VUs  0m31.9s/2m0s

running (0m32.9s), 1000/1000 VUs, 5557 complete and 0 interrupted iterations
default   [  27% ] 1000 VUs  0m32.9s/2m0s

running (0m33.9s), 1000/1000 VUs, 5737 complete and 0 interrupted iterations
default   [  28% ] 1000 VUs  0m33.9s/2m0s

running (0m34.9s), 1000/1000 VUs, 5908 complete and 0 interrupted iterations
default   [  29% ] 1000 VUs  0m34.9s/2m0s

running (0m35.9s), 1000/1000 VUs, 6081 complete and 0 interrupted iterations
default   [  30% ] 1000 VUs  0m35.9s/2m0s

running (0m36.9s), 1000/1000 VUs, 6262 complete and 0 interrupted iterations
default   [  31% ] 1000 VUs  0m36.9s/2m0s

running (0m37.9s), 1000/1000 VUs, 6435 complete and 0 interrupted iterations
default   [  32% ] 1000 VUs  0m37.9s/2m0s

running (0m38.9s), 1000/1000 VUs, 6623 complete and 0 interrupted iterations
default   [  32% ] 1000 VUs  0m38.9s/2m0s

running (0m39.9s), 1000/1000 VUs, 6790 complete and 0 interrupted iterations
default   [  33% ] 1000 VUs  0m39.9s/2m0s

running (0m40.9s), 1000/1000 VUs, 6948 complete and 0 interrupted iterations
default   [  34% ] 1000 VUs  0m40.9s/2m0s

running (0m41.9s), 1000/1000 VUs, 7123 complete and 0 interrupted iterations
default   [  35% ] 1000 VUs  0m41.9s/2m0s

running (0m42.9s), 1000/1000 VUs, 7297 complete and 0 interrupted iterations
default   [  36% ] 1000 VUs  0m42.9s/2m0s

running (0m43.9s), 1000/1000 VUs, 7470 complete and 0 interrupted iterations
default   [  37% ] 1000 VUs  0m43.9s/2m0s

running (0m44.9s), 1000/1000 VUs, 7635 complete and 0 interrupted iterations
default   [  37% ] 1000 VUs  0m44.9s/2m0s

running (0m45.9s), 1000/1000 VUs, 7832 complete and 0 interrupted iterations
default   [  38% ] 1000 VUs  0m45.9s/2m0s

running (0m46.9s), 1000/1000 VUs, 8016 complete and 0 interrupted iterations
default   [  39% ] 1000 VUs  0m46.9s/2m0s

running (0m47.9s), 1000/1000 VUs, 8171 complete and 0 interrupted iterations
default   [  40% ] 1000 VUs  0m47.9s/2m0s

running (0m48.9s), 1000/1000 VUs, 8361 complete and 0 interrupted iterations
default   [  41% ] 1000 VUs  0m48.9s/2m0s

running (0m49.9s), 1000/1000 VUs, 8533 complete and 0 interrupted iterations
default   [  42% ] 1000 VUs  0m49.9s/2m0s

running (0m50.9s), 1000/1000 VUs, 8706 complete and 0 interrupted iterations
default   [  42% ] 1000 VUs  0m50.9s/2m0s

running (0m51.9s), 1000/1000 VUs, 8879 complete and 0 interrupted iterations
default   [  43% ] 1000 VUs  0m51.9s/2m0s

running (0m52.9s), 1000/1000 VUs, 9052 complete and 0 interrupted iterations
default   [  44% ] 1000 VUs  0m52.9s/2m0s

running (0m53.9s), 1000/1000 VUs, 9227 complete and 0 interrupted iterations
default   [  45% ] 1000 VUs  0m53.9s/2m0s

running (0m54.9s), 1000/1000 VUs, 9402 complete and 0 interrupted iterations
default   [  46% ] 1000 VUs  0m54.9s/2m0s

running (0m55.9s), 1000/1000 VUs, 9557 complete and 0 interrupted iterations
default   [  47% ] 1000 VUs  0m55.9s/2m0s

running (0m56.9s), 1000/1000 VUs, 9723 complete and 0 interrupted iterations
default   [  47% ] 1000 VUs  0m56.9s/2m0s

running (0m57.9s), 1000/1000 VUs, 9897 complete and 0 interrupted iterations
default   [  48% ] 1000 VUs  0m57.9s/2m0s

running (0m58.9s), 1000/1000 VUs, 10080 complete and 0 interrupted iterations
default   [  49% ] 1000 VUs  0m58.9s/2m0s

running (0m59.9s), 1000/1000 VUs, 10251 complete and 0 interrupted iterations
default   [  50% ] 1000 VUs  0m59.9s/2m0s

running (1m00.9s), 1000/1000 VUs, 10423 complete and 0 interrupted iterations
default   [  51% ] 1000 VUs  1m00.9s/2m0s

running (1m01.9s), 1000/1000 VUs, 10599 complete and 0 interrupted iterations
default   [  52% ] 1000 VUs  1m01.9s/2m0s

running (1m02.9s), 1000/1000 VUs, 10774 complete and 0 interrupted iterations
default   [  52% ] 1000 VUs  1m02.9s/2m0s

running (1m03.9s), 1000/1000 VUs, 10939 complete and 0 interrupted iterations
default   [  53% ] 1000 VUs  1m03.9s/2m0s

running (1m04.9s), 1000/1000 VUs, 11122 complete and 0 interrupted iterations
default   [  54% ] 1000 VUs  1m04.9s/2m0s

running (1m05.9s), 1000/1000 VUs, 11296 complete and 0 interrupted iterations
default   [  55% ] 1000 VUs  1m05.9s/2m0s

running (1m06.9s), 1000/1000 VUs, 11493 complete and 0 interrupted iterations
default   [  56% ] 1000 VUs  1m06.9s/2m0s

running (1m07.9s), 1000/1000 VUs, 11668 complete and 0 interrupted iterations
default   [  57% ] 1000 VUs  1m07.9s/2m0s

running (1m08.9s), 1000/1000 VUs, 11845 complete and 0 interrupted iterations
default   [  57% ] 1000 VUs  1m08.9s/2m0s

running (1m09.9s), 1000/1000 VUs, 12018 complete and 0 interrupted iterations
default   [  58% ] 1000 VUs  1m09.9s/2m0s

running (1m10.9s), 1000/1000 VUs, 12189 complete and 0 interrupted iterations
default   [  59% ] 1000 VUs  1m10.9s/2m0s

running (1m11.9s), 1000/1000 VUs, 12354 complete and 0 interrupted iterations
default   [  60% ] 1000 VUs  1m11.9s/2m0s

running (1m12.9s), 1000/1000 VUs, 12518 complete and 0 interrupted iterations
default   [  61% ] 1000 VUs  1m12.9s/2m0s

running (1m13.9s), 1000/1000 VUs, 12692 complete and 0 interrupted iterations
default   [  62% ] 1000 VUs  1m13.9s/2m0s

running (1m14.9s), 1000/1000 VUs, 12865 complete and 0 interrupted iterations
default   [  62% ] 1000 VUs  1m14.9s/2m0s

running (1m15.9s), 1000/1000 VUs, 13032 complete and 0 interrupted iterations
default   [  63% ] 1000 VUs  1m15.9s/2m0s

running (1m16.9s), 1000/1000 VUs, 13204 complete and 0 interrupted iterations
default   [  64% ] 1000 VUs  1m16.9s/2m0s

running (1m17.9s), 1000/1000 VUs, 13384 complete and 0 interrupted iterations
default   [  65% ] 1000 VUs  1m17.9s/2m0s

running (1m18.9s), 1000/1000 VUs, 13569 complete and 0 interrupted iterations
default   [  66% ] 1000 VUs  1m18.9s/2m0s

running (1m19.9s), 1000/1000 VUs, 13742 complete and 0 interrupted iterations
default   [  67% ] 1000 VUs  1m19.9s/2m0s

running (1m20.9s), 1000/1000 VUs, 13903 complete and 0 interrupted iterations
default   [  67% ] 1000 VUs  1m20.9s/2m0s

running (1m21.9s), 1000/1000 VUs, 14071 complete and 0 interrupted iterations
default   [  68% ] 1000 VUs  1m21.9s/2m0s

running (1m22.9s), 1000/1000 VUs, 14247 complete and 0 interrupted iterations
default   [  69% ] 1000 VUs  1m22.9s/2m0s

running (1m23.9s), 1000/1000 VUs, 14412 complete and 0 interrupted iterations
default   [  70% ] 1000 VUs  1m23.9s/2m0s

running (1m24.9s), 1000/1000 VUs, 14609 complete and 0 interrupted iterations
default   [  71% ] 1000 VUs  1m24.9s/2m0s

running (1m25.9s), 1000/1000 VUs, 14781 complete and 0 interrupted iterations
default   [  72% ] 1000 VUs  1m25.9s/2m0s

running (1m26.9s), 1000/1000 VUs, 14959 complete and 0 interrupted iterations
default   [  72% ] 1000 VUs  1m26.9s/2m0s

running (1m27.9s), 1000/1000 VUs, 15135 complete and 0 interrupted iterations
default   [  73% ] 1000 VUs  1m27.9s/2m0s

running (1m28.9s), 1000/1000 VUs, 15299 complete and 0 interrupted iterations
default   [  74% ] 1000 VUs  1m28.9s/2m0s

running (1m29.9s), 1000/1000 VUs, 15474 complete and 0 interrupted iterations
default   [  75% ] 1000 VUs  1m29.9s/2m0s

running (1m30.9s), 1000/1000 VUs, 15649 complete and 0 interrupted iterations
default   [  76% ] 1000 VUs  1m30.9s/2m0s

running (1m31.9s), 1000/1000 VUs, 15826 complete and 0 interrupted iterations
default   [  77% ] 1000 VUs  1m31.9s/2m0s

running (1m32.9s), 1000/1000 VUs, 16000 complete and 0 interrupted iterations
default   [  77% ] 1000 VUs  1m32.9s/2m0s

running (1m33.9s), 1000/1000 VUs, 16176 complete and 0 interrupted iterations
default   [  78% ] 1000 VUs  1m33.9s/2m0s

running (1m34.5s), 1000/1000 VUs, 16265 complete and 0 interrupted iterations
default   [  79% ] 1000 VUs  1m34.5s/2m0s

running (1m34.5s), 1000/1000 VUs, 16267 complete and 0 interrupted iterations
default   [  79% ] 1000 VUs  1m34.5s/2m0s

running (1m34.5s), 1000/1000 VUs, 16272 complete and 0 interrupted iterations
default   [  79% ] 1000 VUs  1m34.5s/2m0s

running (1m34.6s), 1000/1000 VUs, 16283 complete and 0 interrupted iterations
default   [  79% ] 1000 VUs  1m34.6s/2m0s

running (1m34.6s), 1000/1000 VUs, 16297 complete and 0 interrupted iterations
default   [  79% ] 1000 VUs  1m34.6s/2m0s

running (1m34.7s), 1000/1000 VUs, 16310 complete and 0 interrupted iterations
default   [  79% ] 1000 VUs  1m34.7s/2m0s

running (1m34.8s), 1000/1000 VUs, 16333 complete and 0 interrupted iterations
default   [  79% ] 1000 VUs  1m34.8s/2m0s

running (1m34.9s), 1000/1000 VUs, 16346 complete and 0 interrupted iterations
default   [  79% ] 1000 VUs  1m34.9s/2m0s

running (1m34.9s), 1000/1000 VUs, 16353 complete and 0 interrupted iterations
default   [  79% ] 1000 VUs  1m34.9s/2m0s

running (1m35.9s), 1000/1000 VUs, 16546 complete and 0 interrupted iterations
default   [  80% ] 1000 VUs  1m35.9s/2m0s

running (1m36.9s), 1000/1000 VUs, 16720 complete and 0 interrupted iterations
default   [  81% ] 1000 VUs  1m36.9s/2m0s

running (1m37.9s), 1000/1000 VUs, 16886 complete and 0 interrupted iterations
default   [  82% ] 1000 VUs  1m37.9s/2m0s

running (1m38.9s), 1000/1000 VUs, 17062 complete and 0 interrupted iterations
default   [  82% ] 1000 VUs  1m38.9s/2m0s

running (1m39.9s), 1000/1000 VUs, 17234 complete and 0 interrupted iterations
default   [  83% ] 1000 VUs  1m39.9s/2m0s

running (1m40.9s), 1000/1000 VUs, 17408 complete and 0 interrupted iterations
default   [  84% ] 1000 VUs  1m40.9s/2m0s

running (1m41.9s), 1000/1000 VUs, 17578 complete and 0 interrupted iterations
default   [  85% ] 1000 VUs  1m41.9s/2m0s

running (1m42.9s), 1000/1000 VUs, 17751 complete and 0 interrupted iterations
default   [  86% ] 1000 VUs  1m42.9s/2m0s

running (1m43.9s), 1000/1000 VUs, 17927 complete and 0 interrupted iterations
default   [  87% ] 1000 VUs  1m43.9s/2m0s

running (1m44.9s), 1000/1000 VUs, 18091 complete and 0 interrupted iterations
default   [  87% ] 1000 VUs  1m44.9s/2m0s

running (1m45.9s), 1000/1000 VUs, 18259 complete and 0 interrupted iterations
default   [  88% ] 1000 VUs  1m45.9s/2m0s

running (1m46.9s), 1000/1000 VUs, 18443 complete and 0 interrupted iterations
default   [  89% ] 1000 VUs  1m46.9s/2m0s

running (1m47.9s), 1000/1000 VUs, 18627 complete and 0 interrupted iterations
default   [  90% ] 1000 VUs  1m47.9s/2m0s

running (1m48.9s), 1000/1000 VUs, 18802 complete and 0 interrupted iterations
default   [  91% ] 1000 VUs  1m48.9s/2m0s

running (1m49.9s), 1000/1000 VUs, 18976 complete and 0 interrupted iterations
default   [  92% ] 1000 VUs  1m49.9s/2m0s

running (1m50.9s), 1000/1000 VUs, 19148 complete and 0 interrupted iterations
default   [  92% ] 1000 VUs  1m50.9s/2m0s

running (1m51.9s), 1000/1000 VUs, 19326 complete and 0 interrupted iterations
default   [  93% ] 1000 VUs  1m51.9s/2m0s

running (1m52.9s), 1000/1000 VUs, 19497 complete and 0 interrupted iterations
default   [  94% ] 1000 VUs  1m52.9s/2m0s

running (1m53.9s), 1000/1000 VUs, 19658 complete and 0 interrupted iterations
default   [  95% ] 1000 VUs  1m53.9s/2m0s

running (1m54.9s), 1000/1000 VUs, 19820 complete and 0 interrupted iterations
default   [  96% ] 1000 VUs  1m54.9s/2m0s

running (1m55.9s), 1000/1000 VUs, 19991 complete and 0 interrupted iterations
default   [  97% ] 1000 VUs  1m55.9s/2m0s

running (1m56.9s), 1000/1000 VUs, 20163 complete and 0 interrupted iterations
default   [  97% ] 1000 VUs  1m56.9s/2m0s

running (1m57.9s), 1000/1000 VUs, 20360 complete and 0 interrupted iterations
default   [  98% ] 1000 VUs  1m57.9s/2m0s

running (1m58.9s), 1000/1000 VUs, 20538 complete and 0 interrupted iterations
default   [  99% ] 1000 VUs  1m58.9s/2m0s

running (1m59.9s), 1000/1000 VUs, 20710 complete and 0 interrupted iterations
default   [ 100% ] 1000 VUs  1m59.9s/2m0s

running (2m00.9s), 0845/1000 VUs, 20881 complete and 0 interrupted iterations
default ↓ [ 100% ] 1000 VUs  2m0s

running (2m01.9s), 0675/1000 VUs, 21051 complete and 0 interrupted iterations
default ↓ [ 100% ] 1000 VUs  2m0s

running (2m02.9s), 0495/1000 VUs, 21231 complete and 0 interrupted iterations
default ↓ [ 100% ] 1000 VUs  2m0s

running (2m03.9s), 0315/1000 VUs, 21411 complete and 0 interrupted iterations
default ↓ [ 100% ] 1000 VUs  2m0s

running (2m04.9s), 0137/1000 VUs, 21589 complete and 0 interrupted iterations
default ↓ [ 100% ] 1000 VUs  2m0s

     data_received..................: 3.9 MB 31 kB/s
     data_sent......................: 4.2 MB 33 kB/s
     http_req_blocked...............: avg=4.03ms   min=49.9µs   med=347.3µs  max=198.4ms  p(90)=420.26µs p(95)=502.53µs
     http_req_connecting............: avg=3.88ms   min=26.21µs  med=230.11µs max=198.37ms p(90)=283.15µs p(95)=349.25µs
     http_req_duration..............: avg=5.64s    min=76.58ms  med=5.76s    max=6.12s    p(90)=5.93s    p(95)=5.97s   
       { expected_response:true }...: avg=5.64s    min=76.58ms  med=5.76s    max=6.12s    p(90)=5.93s    p(95)=5.97s   
     http_req_failed................: 0.00%  ✓ 0         ✗ 21726 
     http_req_receiving.............: avg=162.97µs min=29.88µs  med=152.17µs max=25.74ms  p(90)=227.2µs  p(95)=302.8µs 
     http_req_sending...............: avg=388.68µs min=9.64µs   med=73.77µs  max=61.25ms  p(90)=109.43µs p(95)=142.08µs
     http_req_tls_handshaking.......: avg=0s       min=0s       med=0s       max=0s       p(90)=0s       p(95)=0s      
     http_req_waiting...............: avg=5.64s    min=70.5ms   med=5.76s    max=6.12s    p(90)=5.93s    p(95)=5.97s   
     http_reqs......................: 21726  172.87559/s
     iteration_duration.............: avg=5.65s    min=127.73ms med=5.76s    max=6.12s    p(90)=5.93s    p(95)=5.97s   
     iterations.....................: 21726  172.87559/s
     vus............................: 137    min=137     max=1000
     vus_max........................: 1000   min=1000    max=1000


running (2m05.7s), 0000/1000 VUs, 21726 complete and 0 interrupted iterations
default ✓ [ 100% ] 1000 VUs  2m0s
